component/policies.jsonnet

```
@@ -0,0 +1,23 @@
+local kyverno = import '../lib/kyverno.libsonnet';
+local common = import 'common.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+
+local params = inv.parameters.kyverno;
+
+local clusterpolicies = std.map(
+  function(name) kyverno.ClusterPolicy(name) {
+    metadata+: {
+      name: name,
+      labels+: common.Labels,
+    },
+  } + com.makeMergeable(params.clusterpolicies[name]),
+  std.objectFields(params.clusterpolicies),
+);
+
+
+{
+  '80_policies': clusterpolicies,
```

I'd use the following to avoid emitting an empty file if no policies are configured:

```suggestion
  [if std.length(clusterpolicies) > 0 then '80_policies']: clusterpolicies,
```